packages/components/src/components/field/field.tsx

```diff
@@ -0,0 +1,21 @@
+import classNames from 'classnames';
+import React, { ReactNode } from 'react';
+
+type TFieldProps = {
+    className?: string;
+    message: string | ReactNode | boolean;
+    type: string;
```

Comment:
```suggestion
    type?: 'error' | 'warn';
```
@iman-fs Not sure tho, Please check 🙏🏻

Reply:
checked it. we can add this change. thanks